mirdata/jams_utils.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""functions for converting mirdata annotations to jams format
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import jams
+
+def jams_converter(
+        beat_data=None,
+        chord_data=None,
+        note_data=None,
+        f0_data=None,
+        section_data=None,
+        key_data=None,
+        artist=None,
+        duration=None,
+        title=None,
+        annotator=None):
```

Comment:
I guess there's going to be more than this as things grow. for now, maybe we should have a `sandbox` variable where we throw any extra information.

Reply:
I'm not sure what to do with this, in particular:
1. should we include in the `jams` object **all** the metadata we have per track? 
2. should we have some "mandatory" fields such as `artist` and `duration` and leave the rest to a sandbox, or put everything in the sandbox directly?